Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -78,3 +105,28 @@ struct SafeBalancesRequest: JSONRequest {
         }
     }
 }
+
+struct TransactionsRequest: JSONRequest {
+    let address: String
+    let limit: Int = 100
+    let offset: Int = 0
+    var httpMethod: String { return "GET" }
+    var urlPath: String { return "/api/v1/safes/\(address)/all-transactions/" }
+
+    typealias ResponseType = Response
+
+    init(address: String) {
+        self.address = address
+    }
+
+    var query: String? {
+        return "limit=\(limit)&offset=\(offset)"
+    }
+
+    struct Response: Decodable {
```

Comment:
can we reuse 'PagedResponse' from the SafeRelayService file?

Reply:
yes, the `PagedResponse` should be used here, too